.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: minimal
+os: linux
+
+dist: bionic

```

Comment:
we probably should update this as `bionic` is 14.04; 18.04 or 20.04 would be preferable. If it's too much work we can come back as a follow-up (in which case file an issue with this repo).

Reply:
`Bionic Beaver` is 18.04 LTS. 14.04 LTS is `Trusty Tahr`

Reply:
thanks for the correction.